Use consistent style

f-bader opened the pull request:
Use consistent style for all Azure Automation Asset pages including standardized wording for Activities

azurecla commented:
Hi __@noopman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
@dstrockis - can you review when you have a chance?  Thanks.

dstrockis commented:
@BryanLa please don't merge yet, I would need to update the quick start code and test it.

BryanLa commented:
@dstrockis - sounds good

BryanLa commented:
@dstrockis  - just checking back on this one ...

BryanLa commented:
@dstrockis - another ping ...

BryanLa commented:
@dstrockis - ping

dstrockis commented:
Good to go @BryanLa sorry for the delay

BryanLa commented:
@dstrockis - thanks! 
@noopman - thanks for the contribution!

f-bader committed:
Use consistent style

Use consistent style for all Azure Automation Asset pages including standardized wording for Activities

f-bader committed:
Use consistent style

Use consistent style for all Azure Automation Asset pages including standardized wording for Activities

f-bader committed:
Use consistent style

Use consistent style for all Azure Automation Asset pages including standardized wording for Activities

f-bader committed:
Use consistent style

Use consistent style for all Azure Automation Asset pages including standardized wording for Activities

f-bader committed:
Merge pull request #1 from f-bader/patch-7

Use consistent style

f-bader committed:
Merge pull request #2 from f-bader/patch-8

Use consistent style
